apolloschurchapp/src/tabs/Tab.js

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { useApolloClient, gql, useQuery } from '@apollo/client';
+
+import { BackgroundView, NavigationService } from '@apollosproject/ui-kit';
+import {
+  FeaturesFeedConnected,
+  FEATURE_FEED_ACTION_MAP,
+  RockAuthedWebBrowser,
+} from '@apollosproject/ui-connected';
+
+import { checkOnboardingStatusAndNavigate } from '@apollosproject/ui-onboarding';
+import { createNativeStackNavigator } from 'react-native-screens/native-stack';
+
+import { ONBOARDING_VERSION } from '../ui/Onboarding';
+
+function handleOnPress({ action, ...props }) {
+  if (FEATURE_FEED_ACTION_MAP[action]) {
+    FEATURE_FEED_ACTION_MAP[action]({ action, ...props });
+  }
+}
+
+export const createTab = ({ tabName, screenOptions, ...props }) => {
+  const TabStack = createNativeStackNavigator();
+  const TabComponent = () => <Tab tab={tabName.toUpperCase()} />;
```

Comment:
Could this be an argument to the component somehow?

Reply:
yah, made an explicit prop